compiler/rustc_metadata/src/errors.rs

```diff
@@ -45,7 +45,7 @@ pub struct CrateDepMultiple {
 #[derive(Subdiagnostic)]
 #[note(metadata_crate_dep_not_static)]
 pub struct NonStaticCrateDep {
-    pub crate_name: Symbol,
+    pub crate_name_: Symbol,
```

Comment:
leave a comment here to state that this crate name is different from the root crate